app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -8,13 +8,15 @@ def initialize(identity)
   end
 
   def user_info
-    email = identity.user.email
+    user = identity.user
 
     {
       sub: identity.uuid,
       iss: root_url,
-      email: email,
-      email_verified: email
+      email: user.email,
+      email_verified: true,
+      phone_number: user.phone,
```

Comment:
for the `AttributeAsserter` we chose to not expose the 2FA phone value, but only the verified phone number. We should be consistent.